scripts/driver/driver_internal.sh

```diff
@@ -56,22 +55,32 @@ echo "$parameters" \
   | json2yaml \
 > /data/values.yaml
 
-# Extract name, namespace, and api version.
-name="$(print_config.py \
+# Compose test id.
+test_id="$(cat /dev/urandom \
+    | tr -dc 'a-z0-9' \
+    | head -c 8)"
+
+# Extract keys for name and namespace.
+name_key="$(extract_schema_key.py \
     --schema_file=/data/schema.yaml \
-    --values_file=/data/values.yaml \
-    --param='{"x-google-marketplace": {"type": "NAME"}}')"
+    --type NAME)"
 namespace_key="$(extract_schema_key.py \
     --schema_file=/data/schema.yaml \
     --type NAMESPACE)"
 
-# use base64 for BSD systems where tr won't handle illegal characters
-export NAMESPACE="apptest-$(cat /dev/urandom \
-    | base64 \
-    | tr -dc 'a-z0-9' \
-    | fold -w 8 \
-    | head -n 1)"
-export NAME="$name"
+export NAME="name-$test_id"
+export NAMESPACE="namespace-$test_id"
```

Comment:
This namespace doesn't tell what the namespace is used for. Please keep apptest or an equivalent naming.

Reply:
Done.